torchtune/modules/loss/kd_losses.py

```diff
@@ -78,7 +78,23 @@ def __init__(self, num_output_chunks: int = 8, ignore_index: int = -100):
         super().__init__()
         self.num_output_chunks = num_output_chunks
         self.ignore_index = ignore_index
-        self.fkl_loss = ForwardKLLoss(ignore_index)
+
+    def non_chunked_forward_kl_loss(
```

Comment:
Why did you opt to re-create the `ForwardKLLoss` within the function instead of as another class that just returns the unnormalized calculation? I see a couple of options:

1. Re-use the `ForwardKLLoss` method but add a param called `normalize`, which will normalize the loss by non masked tokens
2. Create a new class called `ForwardKLLossUnnormalized` that returns the loss directly to be used in this implementation

All of these options assume we have the need to keep the original `ForwardKLLoss`. I don't know enough to say that we don't, but figured I'd ask if we might want to keep it or if we can just delete.